src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -536,28 +538,33 @@ namespace AppInstaller::CLI::Workflow
             Workflow::ReportExecutionStage(ExecutionStage::PostExecution) <<
             Workflow::ReportARPChanges <<
             Workflow::RecordInstall <<
-            Workflow::RemoveInstaller << 
+            Workflow::RemoveInstaller <<
             Workflow::DisplayInstallationNotes;
     }
 
-    void DownloadSinglePackage(Execution::Context& context)
+    void InstallDependencies(Execution::Context& context)
     {
-        // TODO: Split dependencies from download flow to prevent multiple installations.
+        if (Settings::User().Get<Settings::Setting::InstallIgnoreDependencies>() || context.Args.Contains(Execution::Args::Type::SkipDependencies))
+        {
+            context.Reporter.Warn() << Resource::String::SkippingDependenciesMessage << std::endl;
+            return;
+        }
+
         context <<
-            Workflow::ReportIdentityAndInstallationDisclaimer <<
-            Workflow::ShowPromptsForSinglePackage(/* ensureAcceptance */ true) <<
             Workflow::GetDependenciesFromInstaller <<
             Workflow::ReportDependencies(Resource::String::InstallAndUpgradeCommandsReportDependencies) <<
             Workflow::EnableWindowsFeaturesDependencies <<
-            Workflow::ManagePackageDependencies(Resource::String::InstallAndUpgradeCommandsReportDependencies) <<
-            Workflow::DownloadInstaller;
+            Workflow::ManagePackageDependencies(Resource::String::InstallAndUpgradeCommandsReportDependencies);
     }
 
     void InstallSinglePackage(Execution::Context& context)
     {
         context <<
             Workflow::CheckForUnsupportedArgs <<
-            Workflow::DownloadSinglePackage <<
+            Workflow::ReportIdentityAndInstallationDisclaimer <<
+            Workflow::ShowPromptsForSinglePackage(/* ensureAcceptance */ true) <<
+            Workflow::InstallDependencies <<

```

Comment:
If `--force` is provided but not `--skip-dependencies` and a dependency fails to install, will the context terminate or will it be a warning that a dependency failed to install, but force was specified so execution is continuing?

Reply:
As of right now, the `--force` argument does not affect the package dependencies. It will just terminate if one of the package dependencies fails to install. `SkipDependencies` is a way to completely skip all of the dependencies for a package. I haven't put too much thought into how `--force` will play a role in this (future PR) but it would warn the user that a dependency failed to install, but continue attempting to install the rest of the dependencies as well as the main package.